docs/manuals/gui/viselements/date.md_template

```diff
@@ -15,18 +25,86 @@ The [Stylekit](../styling/stylekit.md) provides a specific class that you can us
 
 # Usage
 
+## Using only the date
+
+Assuming a variable *date* contains a Python `datetime` object:
+```python
+import datetime
+
+date = datetime.datetime(1789, 7, 14, 17, 5, 12)
+```
+
+You can create a date selector that represents it with the following definition:
+
+!!! taipy-element
+    default={date}
```

Comment:
default ? or date ?

Reply:
or is it some magic to generate the right example in different formats ?

Reply:
Indeed there is.